### PR TITLE
{lib}[GCCcore/8.3.0] YACS v0.1.8 w/ Python 2.7.16 + 3.7.4

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegelbot please test @ generoso

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/38c01718179d6a8a889abdb986d037ca for a full test report.

**Comment by akesandgren** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/y/YACS/YACS-0.1.8-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+use_pip = True
+download_dep_fail = True
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```
Comment (Micket) [inline easybuild/easyconfigs/y/YACS/YACS-0.1.8-GCCcore-10.2.0.eb:35]: This can be dropped right

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/y/YACS/YACS-0.1.8-GCCcore-10.2.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/y/YACS/YACS-0.1.8-GCCcore-10.2.0.eb:35]: Ah, right, forgot about that.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/616d99d2926768cac63ad6fe7d6ad296 for a full test report.

**Comment by akesandgren** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/94cb2c1c52b27afee65e847e3207e604 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @akesandgren!

**Pull request merged**